internal/cli/atlas/quickstart/confirm_cluster_setup.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quickstart
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+func (opts *Opts) askConfirmConfigQuestion() error {
+	if opts.Confirm {
+		return nil
```

Comment:
[q] when is this condition reached? do we have a flag?

Reply:
my point here is, are going to add a flag to the command? like `--force` or `--confirm`?

Reply:
Got it. Let me add that quickly!